generators/entity/index.js

```diff
@@ -219,13 +219,14 @@ module.exports = class extends BaseGenerator {
             validateTableName() {
                 const prodDatabaseType = this.prodDatabaseType;
                 const entityTableName = this.entityTableName;
+                const jhiPrefix = this.jhiPrefix;
```

Comment:
make sure the prefix is in table name format hence introduce a new param 
```
this.jhiTablePrefix = this.getTableName(this.jhiPrefix);`
```
per sub generator and then use that instead of ` this.jhiPrefix`

Reply:
You didn't do this yet?

Reply:
I make the requested changes and commit them. Unfrotunately I overide the first commit by ammending the last commit and now the history of the PR is not clean anymore. Sorry for that. If there is something else for this PR to do let me know. Tank you!